File: platform/control-centre/src/test/scala/strata/server/lobby/controlcentre/SpringContextLoadsIntegrationTest.scala
package strata.server.lobby.controlcentre

import org.scalatest.matchers.ShouldMatchers
import org.scalatest.FlatSpec
import org.springframework.context.support.ClassPathXmlApplicationContext
import org.scalatest.junit.JUnitRunner
import org.junit.runner.RunWith

@RunWith(classOf[JUnitRunner])
class SpringContextLoadsIntegrationTest extends FlatSpec with ShouldMatchers {

    "The Spring Context" should "load without error" in {
        val context = new ClassPathXmlApplicationContext("classpath:/META-INF/spring/root-handler-spring.xml");

        context.getBean("velocityViewResolver") should not be (null)
    }

}

File: platform/platform-worker/src/test/java/com/yazino/platform/SpringContextLoadsIntegrationTest.java
package com.yazino.platform;

import org.junit.Test;
import org.springframework.context.support.ClassPathXmlApplicationContext;
import org.springframework.test.annotation.DirtiesContext;

public class SpringContextLoadsIntegrationTest {

    @Test
    @DirtiesContext
    public void testSpringContextLoadsCorrectly() {
        new ClassPathXmlApplicationContext("classpath:/META-INF/spring/context.xml");
    }

}
